geonode/geoserver/geofence.py

```diff
@@ -310,73 +340,127 @@ def insert_rule(self, rule: Rule):
             http://<host>:<port>/geoserver/rest/geofence/rules
             """
             r = requests.post(
-                f'{self.baseurl.rstrip("/")}/geofence/rules',
-                # headers={'Content-type': 'application/json'},
+                f"{self.baseurl}rules",
                 json=rule.get_object(),
                 auth=HTTPBasicAuth(self.username, self.pw),
-                timeout=ogc_server_settings.get("TIMEOUT", 60),
+                timeout=self.timeout,
                 verify=False,
             )
 
             if r.status_code not in (200, 201):
-                logger.debug(f"Could not insert rule: [{r.status_code}] - {r.content}")
-                raise GeofenceException(f"Could not insert rule: [{r.status_code}]")
+                logger.debug(f"Could not insert rule: [{r.status_code}] - {r.text}")
```

Comment:
Since its raising an exception, it might be better to increase this log level from debug or remove it since is raising the exception

Reply:
Usually the detailed information should be logged when debugging.
It's not said that the exception message is displayed at higher level.